src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -267,28 +269,46 @@ def load_scaling_factor(
 def load_anc_proportion(
     key: str, location: str, years: Optional[Union[int, str, list[int]]] = None
 ) -> pd.DataFrame:
-    anc_proportion = load_standard_data(key, location, years)
-    year_start, year_end = 2021, 2022
-    lower_value = anc_proportion.loc[(year_start, year_end, "lower_value"), "value"]
-    mean_value = anc_proportion.loc[(year_start, year_end, "mean_value"), "value"]
-    upper_value = anc_proportion.loc[(year_start, year_end, "upper_value"), "value"]
-
-    try:
-        anc_proportion_dist = sampling.get_truncnorm_from_quantiles(
-            mean=mean_value, lower=lower_value, upper=upper_value
+    if key == data_keys.ANC.ANCfirst:
+        data = pd.read_csv(paths.ANC_DATA_DIR / "anc_first.csv")
+        location_id = utility_data.get_location_id(location)
+        data = data.loc[data["location_id"] == location_id]
+        data = data.loc[data["year_id"] == metadata.ARTIFACT_YEAR_START].rename(
+            {"year_id": "year_start"}, axis=1
         )
-        anc_proportion_draws = anc_proportion_dist.rvs(data_values.NUM_DRAWS).reshape(
-            1, data_values.NUM_DRAWS
+        data["year_end"] = metadata.ARTIFACT_YEAR_END
+        data = data.drop(
+            ["age_group_id", "sex_id", "location_id", "mean", "lower", "upper"], axis=1
         )
-    except FloatingPointError:
-        print("FloatingPointError encountered, proceeding with caution.")
-        anc_proportion_draws = np.full((1, data_values.NUM_DRAWS), mean_value)
-
-    draw_columns = [f"draw_{i:d}" for i in range(data_values.NUM_DRAWS)]
-    anc_proportion_draws_df = pd.DataFrame(anc_proportion_draws, columns=draw_columns)
-    anc_proportion_draws_df["year_start"] = year_start
-    anc_proportion_draws_df["year_end"] = year_end
-    return anc_proportion_draws_df.set_index(["year_start", "year_end"])
+        data = data.set_index(["year_start", "year_end"])
+        return data
+    elif key == data_keys.ANC.ANC1 or key == data_keys.ANC.ANC4:
+        anc_proportion = load_standard_data(key, location, years)
+        year_start, year_end = metadata.ARTIFACT_YEAR_START, metadata.ARTIFACT_YEAR_END
+        lower_value = anc_proportion.loc[(year_start, year_end, "lower_value"), "value"]
+        mean_value = anc_proportion.loc[(year_start, year_end, "mean_value"), "value"]
+        upper_value = anc_proportion.loc[(year_start, year_end, "upper_value"), "value"]
+
+        try:
+            anc_proportion_dist = sampling.get_truncnorm_from_quantiles(
+                mean=mean_value, lower=lower_value, upper=upper_value
+            )
+            anc_proportion_draws = get_random_variable_draws(
+                metadata.ARTIFACT_COLUMNS, key, anc_proportion_dist
+            )
+        except FloatingPointError:
+            print("FloatingPointError encountered, proceeding with caution.")
```

Comment:
I know this is old logic, but maybe add a comment saying exactly what you're doing. I'm not really following...it's simply using only mean_value rather than the distribution? Is that really correct?